app/src/main/java/de/danoeh/antennapod/view/ChapterSeekBar.java

```diff
@@ -59,6 +62,17 @@ public void setDividerPos(final float[] dividerPos) {
         }
     }
 
+    public void highlightCurrentChapter() {
+        isEnabled = true;
```

Comment:
Could you please rename the variable? I think it is a bit misleading. I would suggest `isHighlighted`.

Reply:
Fixed :)